lib/wax/application.ex

```diff
@@ -6,7 +6,7 @@ defmodule Wax.Application do
 
   def start(_type, _args) do
     children = [
-      worker(Wax.Metadata, [])
```

Comment:
Before, when using the token, the service would process metadata update only when the token was set. Now, it would be nice to have an option allowing to disable metadata updating. When attestation is not used (`"none"`), metadata is not needed.